data/xml/W19.xml

```diff
@@ -1553,4 +1553,4215 @@
+    <publisher>Association for Computational Linguistics</publisher>
+    <pages>223–233</pages>
+    <abstract>We tackle the important task of part-of-speech tagging using a neural model in the zero-resource scenario, where we have no access to gold-standard POS training data. We compare this scenario with the low-resource scenario, where we have access to a small amount of gold-standard POS training data. Our experiments focus on Ukrainian as a representative of under-resourced languages. Russian is highly related to Ukrainian, so we exploit gold-standard Russian POS tags. We consider four techniques to perform Ukrainian POS tagging: zero-shot tagging and cross-lingual annotation projection (for the zero-resource scenario), and compare these with self-training and multilingual learning (for the low-resource scenario). We find that cross-lingual annotation projection works particularly well in the zero-resource scenario.</abstract>
+    <url>http://www.aclweb.org/anthology/W19-1425</url>
+    <bibtype>inproceedings</bibtype>
+    <bibkey>huck-dutka-fraser:2019:W19-14</bibkey>
+  </paper>
+
+  <paper id="1500">
+    <title>Proceedings of the Third Workshop on Structured Prediction for <fixed-case>NLP</fixed-case></title>
+    <editor><first>Andre</first><last>Martins</last></editor>
+    <editor><first>Andreas</first><last>Vlachos</last></editor>
+    <editor><first>Zornitsa</first><last>Kozareva</last></editor>
+    <editor><first>Sujith</first><last>Ravi</last></editor>
+    <editor><first>Gerasimos</first><last>Lampouras</last></editor>
+    <editor><first>Vlad</first><last>Niculae</last></editor>
+    <editor><first>Julia</first><last>Kreutzer</last></editor>
+    <month>June</month>
+    <year>2019</year>
+    <address>Minneapolis, Minnesota</address>
+    <publisher>Association for Computational Linguistics</publisher>
+    <url>http://www.aclweb.org/anthology/W19-15</url>
+    <bibtype>book</bibtype>
+    <bibkey>W19-15:2019</bibkey>
+  </paper>
+
+  <paper id="1501">
+    <title>Parallelizable Stack Long Short-Term Memory</title>
+    <author><first>Shuoyang</first><last>Ding</last></author>
+    <author><first>Philipp</first><last>Koehn</last></author>
+    <booktitle>Proceedings of the Third Workshop on Structured Prediction for <fixed-case>NLP</fixed-case></booktitle>
+    <month>June</month>
+    <year>2019</year>
+    <address>Minneapolis, Minnesota</address>
+    <publisher>Association for Computational Linguistics</publisher>
+    <pages>1–6</pages>
+    <abstract>Stack Long Short-Term Memory (StackLSTM) is useful for various applications such as parsing and string-to-tree neural machine translation, but it is also known to be notoriously difficult to parallelize for GPU training due to the fact that the computations are dependent on discrete operations. In this paper, we tackle this problem by utilizing state access patterns of StackLSTM to homogenize computations with regard to different discrete operations. Ou
```

Comment:
This line produces a warning during the build due to the usage of `\mathit`. That means it'll show the raw TeX command on the website. I should probably add some code to handle this in `texmath.py`, it's pretty trivial...